sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue

```diff
@@ -41,9 +41,24 @@
         <c-o-e-summary-data
           :locationId="locationId"
           :enrollmentPeriod="EnrollmentPeriod.Upcoming"
-          header="Upcoming enrolment"
-          subTitle="These applications are still outside of the 21 days of the study start date."
-        />
+          header="Upcoming and
+        Previous Enrolments"
+        >
+          <template #coeSummarySubtitle>
+            <span>This page contains two types of records:</span>
+            <ul>
+              <li>
+                Disbursements requests which are upcoming and will be required
+                at a later date. These are indicated by a 'Required' status.
```

Comment:
Asked a minor question to devs about styling with single quotes.

Reply:
@CarlyCotton has confirmed to use **bold** as it is used elsewhere in the application.